pm_muairss_write/pm_muairss_write.xml

```diff
@@ -3,27 +3,43 @@
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
         pip show pymuon-suite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
-        cp $structure input.cell &&
-        cp $params params.yaml &&
+        ln -s $structure input.cell &&
+        ln -s $params params.yaml &&
+        #if str($iscastep.iscastep_sel)=="yes":
+            ln -s $castep_param castep.param &&
+        #end if
         pm-muairss -t w input.cell params.yaml &&
-        zip -r out_zip.zip $out_folder &&
-        find $out_folder > tree.txt
+        out_folder="`python ${__tool_directory__}/get_out_folder.py`" &&
+        #if str($iscastep.iscastep_sel)=="yes":
+            zip -r out_zip1.zip \$out_folder params.yaml input.cell castep.param &&
+        #else if str($iscastep.iscastep_sel)=="no":
+            zip -r out_zip1.zip \$out_folder params.yaml input.cell &&
+        #end if
+        ln -s out_zip1.zip out_zip.zip &&
+        find \$out_folder > tree.txt
     ]]></command>
     <inputs>
         <param type="data" name="structure" format="cell" />
```

Comment:
These parameters could all do with help text etc., but this applies to all tools and I think we can make it a separate issue.